### PR TITLE
fix(Button): fixes padding and colors styles

### PR DESCRIPTION
**Pull request by renatogalvones**
## Description
https://jirasoftware.catho.com.br/browse/QTM-60

## Setup
to view the components behavior, run `yarn storybook`
Look for `Button`

## Review guide
- [ ] Coverage (coverage status should not regress)\
      - `yarn test:coverage`
- [x] Unit tests (yarn test:components)
- [x] Regression \
      - first start the storybook for regression tests(and keep it open): ` yarn test:regression:storybook`; \
      - then run the regression tests: `yarn test:regression`
- [x] Code review

### Browsers review
- [x] Layout review
  - [x] Edge
  - [x] IE 11
  - [x] Firefox
  - [x] Chrome
  - [x] Safari
  - [x] Mobile
- [x] Ux review validation

### CODE REVIEW

File: stories/Button/Button.regression-test.story.jsx [hunk 1]
```diff
@@ -3,17 +3,73 @@ import { storiesOf } from '@storybook/react';
...
+
+  .add('primary full', () => <Button full>Catho</Button>)
+  .add('primary icon info', () => <Button icon="info">Catho</Button>)
+  .add('primary size xsmall', () => <Button size="xsmall">Catho</Button>)
+  .add('primary size small', () => <Button size="small">Catho</Button>)
+  .add('primary size large', () => <Button size="large">Catho</Button>)
+  .add('primary size xlarge', () => <Button size="xlarge">Catho</Button>)
+
+  .add('secondary default', () => <Button skin="secondary">Catho</Button>)
+  .add('secondary disabled', () => (
```
Comment (rapahaeru) [inline stories/Button/Button.regression-test.story.jsx]: how about to keep the pattern in one line on tests, if possible?
Just link the primary?
was this made by linter?

File: components/Button/Button.jsx [hunk 2]
```diff
@@ -31,22 +31,24 @@ const ButtonIcon = styled(Icon)`
 `;
 
 const buttonFontAndLineProps = ({ size, theme: { baseFontSize } }) => {
-  const sizes = {
+  const fontSizes = {
```
Comment (rapahaeru) [inline components/Button/Button.jsx]: Show de bola! 👌

File: components/Button/Button.jsx [hunk 3]
```diff
@@ -56,7 +58,7 @@ const StyledButton = styled.button`
   display: flex;
   font-weight: bold;
   justify-content: center;
-  letter-spacing: 0.2px;
+  letter-spacing: 0.24px;
```
[the remark this answers is not in the document]
Comment (pedrohmorais) [reply components/Button/Button.jsx]: This value exists in a lot of other components in our styleguide. I think this is the default letter spacing for the `Button` component.

[the remark this answers is not in the document]
Comment (renatogalvones) [reply components/Button/Button.jsx]: Arthur, didn't even know about this letter-spacing. He removed it.

File: stories/Button/Button.regression-test.story.jsx (hunk 1 above)
Comment (renatogalvones) [reply stories/Button/Button.regression-test.story.jsx]: I tried to keep them in one line but linter made these changes.

File: components/shared/theme.js [hunk 4]
```diff
@@ -80,6 +80,7 @@ const spacing = {
   xlarge: gutter * 4, // 32
   xxlarge: gutter * 5, // 40
   xxxlarge: gutter * 6, // 48
+  xxxxlarge: gutter * 7, // 56
```
[the remark this answers is not in the document]
Comment (renatogalvones) [reply components/shared/theme.js]: Gonna ask Arthur about it but the `x-large` button on styleguide has 56px height so I think this is expected.

**Pull request merged**